phabfive/maniphest.py

```diff
@@ -81,10 +96,149 @@ def create_from_config(self, config_file, dry_run=False):
+            if ("subscribers" not in ticket_data):
+                break
+
+            if not isinstance(ticket_data["subscribers"], list):
+                log.error(f"data key 'subscribers' must be of list type")
+                return (False, None)
+
+            has_errors = False
+            
+            for subscriber in ticket_data["subscribers"]:
+
+                if subscriber not in user_name_to_id_mapping:
+                    log.error(f"Subscriber user name '{subscriber}' not found in Phabricator instance")
+                    has_errors = True
+
+            if has_errors:
+                log.error(f"Could not find one or more specified users, projects in phabricator instance. All available project names")
+                log.error(list(project_name_to_id_mapping.keys()))
+                log.error(f"All available users names")
+                log.error(list(user_name_to_id_mapping.keys()))
+                return (False, None)
+
+        ### Validate priority is consistent with Phabricator instance, default = TICKET_PRIORITY_NORMAL
+        for index, ticket_data in enumerate(data["tickets"]):
+            if ("priority" not in ticket_data):
+                break
+            if ticket_data["priority"] not in TICKET_PRIORITIES:
+                log.error(f"Priority '{ticket_data['priority']}' not found in Phabricator instance")
+                has_errors = True
+
+            if has_errors:
+                log.error(f"Could not find one or more specified users, projects in phabricator instance. All available project names")
+                log.error(list(project_name_to_id_mapping.keys()))
+                log.error(f"All available users names")
+                log.error(list(user_name_to_id_mapping.keys()))
+                return (False, None)
+        
+        task_id_to_phid_mapping = {}
+
+        ###FIXME: Add parent task
+        for index, ticket_data in enumerate(data["tickets"]):
+            if ("parents" not in ticket_data):
+                break
+
+            if not isinstance(ticket_data["parents"], list):
+                log.error(f"data key 'parents' must be of list type")
+                return (False, None)
+
+            has_errors = False
+
+            constraints = {}
+            constraints["ids"] = []
+            
+            for parent in ticket_data["parents"]:
+                if not re.match(r'^(T).*\d$$', parent):
+                    log.error(f"Task 'id' must start with T")
+                    return (False, None)
+                else:
+                    try:
+                        parentTask = int(parent[1:])
+                    except:
+                        log.error(f"Task 'id' must start with T")
+                        return (False, None)
+                    else:
+                        constraints["ids"].append(parentTask)
+            
+            if len(constraints["ids"]) < 0:
+                    break
```

Comment:
Double indent here

Reply:
This is by design i.e. intended to be this way.

Reply:
Double indent can't be done by design? or are we talking about different things maybe?